### PR TITLE
Add UI

### PR DESCRIPTION
**Pull request by paulinagonzalezc**
## OOP school library: add basic UI ✨

In this project, I created a form of UI for my school library app. This way it can be invoked as an executable and not something you use in IRB exclusively.

* Your console app should behave in the same way as it is in the example
* Create an app.rb file that will serve as your console app entry-point
* In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  # Create a student
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent persmission? [Y/N]: '
+    parent_permission = gets.chomp.downcase == 'y'
+    # print 'Select classroom: '
+    # classroom = gets.chomp
```
Comment (victorokoroji) [inline app.rb]: - [x] Kindly ensure you remove commented code from your codebase so that your code will look clean and readable 👍🏽

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @paulinagonzalezc, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (paulinagonzalezc) [reply app.rb]: Thank you @vickymarz let me remove them!

Comment (rloterh) [reply app.rb]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Paulina,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Interactive console app works as intended!

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**